packaging/pep517_backend/_backend.py

```diff
@@ -8,9 +8,8 @@
 import sys
 import typing as t
 from configparser import ConfigParser
-from contextlib import contextmanager, suppress
-from importlib.metadata import import_module
-from io import StringIO
+from contextlib import contextmanager
+from importlib import import_module
```

Comment:
I'd keep refactoring this separate so it doesn't create confusion about being mandatory for the standalone manpage builder to work..